Sources/Plasma/CoreLib/HeadSpin.h

```diff
@@ -368,7 +368,6 @@ int hsMessageBoxWithOwner(hsWindowHndl owner, const wchar_t* message, const wcha
 #   define hsWFopen(name, mode)     fopen(hsWStringToString(name), hsWStringToString(mode))
 
 #   include <limits.h>
-#   define MAX_PATH PATH_MAX
```

Comment:
We should actually get long path support for local files in, but for now having a working patcher seems like the bigger bug.

Reply:
My preference is definitely to deal with this at the network protocol level and not at the HeadSpin.h level. What we're dealing with is an implementation requirement of the network protocol, not an issue with file paths overall.

We've already got some weird implementation requirements for the network protocol, like using `char16_t` for strings. Let's just define a `kCli2File_FilenameSize = 260` constant in NucleusLib/pnNetProtocol/Private/Protocols/Cli2File/pnNpCli2File.h

Reply:
It looks like pnNbConst.h already has a FIXME around this behavior.

I'm fine splitting things but MAX_PATH is also used a lot of places so it would have to be a careful audit. The MAX_PATH define maybe should also be moved out of pnNbConst.h for Windows as well and into the Headspin header. Then a proper constant for all platforms could be defined in pnNbConst.h instead.

Reply:
`MAX_PATH` is always defined on Windows by system headers, so that one in plNbConst.h should only be affecting Linux/macOS

Reply:
I'm going to try to get the network code shifted over to it's own constant. I'll roll that in later today.

Reply:
Ok, the network code is changed over to a kCli2File_FilenameSize constant. I did a one by one find/replace in all network classes to move it over to the new constant. This work should probably be double checked.